File: exemplos_java/Alvo.java
/*
 * Robot: Alvo
 * 
 * Permanece imóvel no campo de batalha. Útil como alvo para testes.
 */

import java.io.*;
import java.util.*;


class Alvo extends Robot {
	// Enquanto true o robô continua a funcionar
	boolean alive = true;
	
	void start() {
		// Inicia o robô
		init();
        while ( alive ) {
	        noop();
	        try {
	        	Thread.sleep(2000);
	        }
	        catch(Exception e){}
        }
	}
	
	
	public static void main( String[] args ) {
		Alvo robot = new Alvo();
		robot.start();
	}
	
}

File: exemplos_python/Alvo.py
#!/usr/bin/env python
# -*- coding:utf-8 -*-
"""
Robot: Alvo

Permanece imóvel no campo de batalha. Útil como alvo para testes.
"""
from port_module import Robot
from time import sleep

class Alvo( Robot ):
    def __init__( self ):
        Robot.__init__( self )
        self._alive = True
        # Inicia o robot
        self.init()

    def start( self ):
        while ( self._alive ):
            sleep( 2 )
            self.noop()



if ( __name__ == "__main__" ):
    robot = Alvo()
    robot.start()